### PR TITLE
adds a utility script to get rc branches

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: util/jenkins/get-rc-branches.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+if [[ ! -d $repo_basename ]]; then
+    $noop git clone "$repo" "$repo_basename" --mirror
+else
+    $noop cd "/var/tmp/$repo_basename"
+    $noop git fetch
+fi
+
+$noop cd "/var/tmp/$repo_basename"
+if [[ -z $noop ]]; then
+    python -c 'import sys, json; print json.dumps([line.lstrip().rstrip()  for line in sys.stdin])' < <(git branch -a | grep "$filter" | sort -r | head )
```
Comment (feanil) [inline util/jenkins/get-rc-branches.sh]: Why not just use line.strip()?

Comment (feanil) [inline util/jenkins/get-rc-branches.sh]: Also what's up with the `< <`  Don't you need a `$` for the command in parens?

Comment (jarv) [inline util/jenkins/get-rc-branches.sh]: fixed

**Comment by feanil** [comment]
:+1:

**Pull request merged**